reference/forms/types/options/by_reference.rst.inc

```diff
@@ -5,7 +5,7 @@ by_reference
 
 In most cases, if you have a ``name`` field, then you expect ``setName()``
 to be called on the underlying object. In some cases, however, ``setName()``
-may *not* be called. Setting ``by_reference`` ensures that the setter is
+may *not* be called. Setting ``by_reference=false`` ensures that the setter is
 called in all cases.
```

Comment:
Actually in this case the setter `setName` __is__ called in all case. Would be better to make it more explicit too by replacing it by:
```rst
In most cases, if you have an ``author`` type, then you expect ``setAuthor()...``
```